.dwollaci.yml

```diff
@@ -43,7 +43,7 @@ stages:
         nvm install
         npm install -g npm
         npm install -g serverless
-        ENVIRONMENT=devint SKRIPTS_DEPLOYMENT_BUCKET=dwolla-encrypted serverless deploy -v --stage devint
+        ENVIRONMENT=devint SKRIPTS_DEPLOYMENT_BUCKET=dwolla-encrypted npm run deploy
```

Comment:
I don't believe we need to run the `deploy` script as this would run a build that we don't want to run here. We only want the `serverless deploy` command.

Reply:
The problem is that the `serverless deploy` relies on a stack.yml being rendered by CDK to determine what is actually being deployed. Without doing `npm run deploy`, `stack.yml` does not get created and then breaks the deployment all together since that is where it is the logicalId's `ResultQueue` and `ErrorQueue` gets created. You will either have to remove the IAM policy that the serverless file creates that gives access to the queues, or allow this build to happen.

Reply:
Okay, and each deploy stage doesn't have access to the files generated during the build stage?

Reply:
Unfortunately the `stack.yml` files that get generated are on a per-environment basis. Unless those are all compiled in the build stage multiple times, we would have to rerun the commands anyway.